### PR TITLE
Ability to definied a custom Template class

### PR DESCRIPTION
**Pull request by WnP**
This is an alternative fix for #3 which IMHO is preferable.

Let me know your through

### CODE REVIEW

File: djangomako/backends.py [hunk 1]
```diff
@@ -92,6 +92,9 @@ def __init__(self, parameters):
         # particular template URI
         options.setdefault('directories', self.template_dirs)
 
+        self.Template = import_string(options.pop(
```
Comment (iamjazzar) [inline djangomako/backends.py]: Can you rename this to `template_class` instead of `Template`?

**Review by iamjazzar — CHANGES_REQUESTED** [review]
Thanks for your contribution, just one minor comment.

**Comment by WnP** [comment]
@ahmedaljazzar sure, no problem, sounds a good choice

**Review by iamjazzar — APPROVED** [review]

**Comment by iamjazzar** [comment]
Thanks for your contribution @WnP :D

**Comment by WnP** [comment]
@ahmedaljazzar you're welcome :wink: 

Any chance to release a version with this fix on pypi?

**Comment by iamjazzar** [comment]
Released, just check `django-mako v1.1.0`

**Pull request merged**